SOCI/README.md

```diff
@@ -0,0 +1,437 @@
+
+| Feature | SOCI Index V1 | SOCI Index V2 |
+|---------|---------------|---------------|
+| Structure | Separate artifact from container image | Bound to container image via image index |
+| Relationship | Loose coupling through Subject metadata | Direct link through image index |
+| Image Modification | No modification required | Requires image index modification |
+| Management | Challenges with replication, deletion, and consistency | Improved scalability and consistency |
+
+![v1](./v1.jpg) ![v2](./v2.jpg)
+
+## DLC SOCI Supported Frameworks
+
+Deep Learning Containers (DLC) team provides SOCI index for the following images both in private registry and [public gallery](https://gallery.ecr.aws/deep-learning-containers): PyTorch Training (x86, arm64), vLLM (x86, arm64), and SGLang.
+
+### Sample URIs
+
+- **ECR Private Registry** (requires AWS account to login):
+  ```
+  763104351884.dkr.ecr.us-east-1.amazonaws.com/pytorch-training:2.9-cpu-py312-ubuntu22.04-sagemaker-v1-soci
+  ```
+
+- **ECR Public Gallery** (no login required):
+  ```
+  public.ecr.aws/deep-learning-containers/pytorch-training:2.9-cpu-py312-ubuntu22.04-sagemaker-v1-soci
+  ```
+
+### DLC SOCI Tag Introduction
+
+For different tracking purposes, DLC images have several tags for one single image. Taking PyTorch training image as an example, it has six tags. Tags without version or with only major version will be overridden by each patch release and always point to the latest release image, while tags with minor version and timestamp will only attach to one specific image.
+
+**Tags attached to specific image:**
+- `2.9.0-gpu-py312-cu130-ubuntu22.04-ec2-v1.2-2025-12-15-22-03-41`
+- `2.9.0-gpu-py312-cu130-ubuntu22.04-ec2-v1.2`
+
+**Tags always attached to latest image:**
+- `2.9-gpu-py312-ec2`
+- `2.9-gpu-py312-cu130-ubuntu22.04-ec2-v1`
+- `2.9.0-gpu-py312-ec2`
+- `2.9.0-gpu-py312-cu130-ubuntu22.04-ec2`
+
+**Usage Guidelines:**
+- To use a specific image, use tags with timestamp and minor version
+- To always get the latest image with CVE patches, use tags without timestamp
+
+**SOCI-Enabled Tags:**
+
+For SOCI index enabled images, DLC adds `-soci` as a tag suffix. For the above image examples, the SOCI tags are:
+
+**Tags attached to specific image:**
+- `2.9.0-gpu-py312-cu130-ubuntu22.04-ec2-v1.2-2025-12-15-22-03-41-soci`
+- `2.9.0-gpu-py312-cu130-ubuntu22.04-ec2-v1.2-soci`
+
+**Tags always attached to latest image:**
+- `2.9-gpu-py312-ec2-soci`
+- `2.9-gpu-py312-cu130-ubuntu22.04-ec2-v1-soci`
+- `2.9.0-gpu-py312-ec2-soci`
+- `2.9.0-gpu-py312-cu130-ubuntu22.04-ec2-soci`
+
+### Supported Versions
```

Comment:
can we keep supported version information non-verbose? So we don't have to update it at every release. Maybe we can let people check the public gallern tags and identify which version has SOCI index